api/API.md

```diff
@@ -180,15 +186,15 @@ The request MUST NOT contain any body content.
 Response:
 
 The response mime type MUST be "application/x-proto-binary".
-The response MUST contain a message with the entire configuration for the given Device. The body MUST be a protobuf message of type [zconfig.EdgeDevConfig](./zconfig/devconfig.proto).
+The response MUST contain a message with the entire configuration for the given Device. The body MUST be a protobuf message of type [config.EdgeDevConfig](./proto/config/devconfig.proto).
 The body MUST contain the entire configuration for the Device.
 The body MUST contain the UUID for the Device on each and every request. The Controller MUST NOT assume that the Device already has the UUID.
 
 #### Arbitrary Config Variables
 
 The `config` endpoint, specifically the `EdgeDevConfig` message, supports arbitrary key/value pairs. These are intended to send implementation-specific configuration to a Device. For example, it might control the frequency of downloading configs, enable debug logging or enable a USB port.
 
-The `EdgeDevConfig` message can contain zero, one or more `ConfigItem` entries, each of which is an arbitrary key/value pair. These SHOULD be used to send implementation-specific configuration to a Device, other than configuration already defined in this API or the messages.
+The `EdgeDevConfig` message can contain zero, one or more `ConfigItem` entries, each of which is an arbitrary key/value pair. These SHOULD be used to send implementation-specific configuration to a Device, other than configuration already defined in this API or the messages. The items are defined in [global-config-variables](../pkg/pillar/docs/global-config-variables.md)
```

Comment:
The only question I have is around the definition of `global-config-variables` defined in `pkg/pillar` vs here (or somewhere independent). Is the list of config variables implemented-dependent (i.e. not an official part of the API, but part of the EVE reference implementation)? If so, then we probably should say here something like, "the individual valid key/value pairs are implementation-dependent. The reference implementation EVE contains a set available at <url>." On the other hand, if it is an official part of the API, then we should include it in this directory, along with a clearly-defined mechanism for adding custom ones that are not referenced here, e.g. "x-<key>" or something similar.